.github/workflows/tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
I suggest only running one or two of the latest Python versions. Also 3.6 was recently EOL’d anyway so it should be taken out.

Reply:
Thanks! I wanted to ask about 3.6.
If I take it out, should I mention explicitly it's not supported any more?

I guess we could test on 3.7, the oldest supported version, 3.10, the newest one. Is that enough?

Reply:
I was actually going to say just 3.9 and 3.10. I wonder if there is a “latest” we can use instead of always manually chasing the latest version. That and a “next” that runs but doesn’t block if it fails so we are at least made aware. 

The PyPI classifiers in setup.py should be checked to make sure they are consistent with this too, or just left generic with only python3.

Reply:
Sorry I didn’t answer you question. Yes I would mention in the change log which versions are supported or dropped.

Reply:
If you want to just do 3.7 and 3.10 that is fine with me too. I primarily care that the latest version is tested. Secondary concern is that we aren’t spending too much time on failures with old versions no one is even using. Or if they are they could just upgrade to a newer version. But I’ll leave it up to you since you may have different requirements.